pkg/config/config.go

```diff
@@ -203,3 +184,31 @@ type PrinterConfig struct {
 	ContainerMode ContainerMode
 	RelativeTS    bool
 }
+
+// PoliciesConfig holds the Tracee configuration required by the Policies computation.
+type PoliciesConfig struct {
+	DNSCacheConfig   bool
+	ProcTreeSource   proctree.SourceType
+	CaptureExec      bool
```

Comment:
I am not sure what you did here to be honest.
Why not just use the `CaptureConfig` struct?
You are using this to create the events state, but the internal configs are necessary to track also when changed by policies, no?
And if you prefer to avoid dependencies between this config and the `CaptureConfig`, I think it will look cleaner to have another capture config here that include a bool for each capture type.

Reply:
Oh, I got you point. Disregarding the OutputPath field config.CaptureConfig can replace this.

Thanks for the tip. I'll recheck this.